src/types.js

```diff
@@ -664,6 +666,13 @@ export type ClassMethod = MethodBase &
     variance?: ?FlowVariance, // TODO: Not in spec
   };
 
+export type ClassPrivateMethod = NodeBase &
+  ClassMethodOrDeclareMethodCommon & {
+    type: "ClassPrivateMethod",
+    key: Identifier,
+    static: false,
```

Comment:
Should be `boolean`, right?

Reply:
Fixed.

Reply:
Oh, it's already in `ClassMethodOrDeclareMethodCommon`, so we can just drop this line.